kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -711,17 +736,16 @@ class ResidualBasedEliminationBuilderAndSolver
             new_max = ceil(0.5*static_cast<double>(old_max));
         }
 
-        DofsArrayType Doftemp;
+        DofsArrayType dof_temp;
         BaseType::mDofSet = DofsArrayType();
 
-        Doftemp.reserve(dofs_aux_list[0].size());
-        for (auto it = dofs_aux_list[0].begin(); it != dofs_aux_list[0].end(); it++)
-        {
-            Doftemp.push_back(it->get());
+        dof_temp.reserve(dofs_aux_list[0].size());
+        for (auto it = dofs_aux_list[0].begin(); it != dofs_aux_list[0].end(); ++it) {
+            dof_temp.push_back(it->get());
         }
-        Doftemp.Sort();
+        dof_temp.Sort();

```

Comment:
I believe you miss adding the dofs which are introduced by the MPCs...

Reply:
This method is override on the MPC version of the B&S

Reply:
i thought we unified this in one class...

ok

Reply:
Not yet, and is not the target of this PR